.circleci/config.yml

```diff
@@ -11,6 +11,6 @@ jobs:
       - run:
           name: test
           command: npm run test
-      - run:
-          name: deploy_staging
-          command: ./scripts/deploy_staging.sh
+      # - run:
```

Comment:
Lol dont just delete, like that masters dont get deployed. Let is run on masters but not on other branches. See circles filtering:
https://circleci.com/docs/2.0/configuration-reference/#filters